frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -120,8 +125,30 @@ export const NodesDisksListPage: React.FC<NodesDisksListPageProps> = ({
   const operatorNs = getNamespace(subscription[0]);
   const csvName = subscription?.[0]?.status?.installedCSV;
   const nodeName = obj.metadata.name;
+  const nodeNameByHostnameLabel = obj.metadata?.labels?.['kubernetes.io/hostname'];
   const nodeRole = getNodeRole(obj);
   const propName = `lvdr-${nodeName}`;
+  const [errorMessage, setError] = React.useState('');
+  const [inProgress, setProgress] = React.useState(false);
+
+  const makeLocalVolumeDiscoverRequest = async (node: string, ns: string) => {
+    setProgress(true);
+    try {
+      await updateLocalVolumeDiscovery([node], ns, setError);
+    } catch (error) {
+      if (error?.response?.status === 404) {
+        try {
+          await createLocalVolumeDiscovery([node], ns, setError);
+        } catch (createError) {
+          setError(createError.message);
+        }
+      } else {
+        setError(error.message);
+      }
+    } finally {
+      setProgress(false);
```

Comment:
If request is successful, we still want to show progress state. 
Once  disks are available table component will render them,
Currently, even after a successful discovery request, we don't get the idea of  disks' loading state.
```suggestion
      if (error?.response?.status === 404) {
        try {
          await createLocalVolumeDiscovery([node], ns, setError);
        } catch (createError) {
          setError(createError.message);
          setProgress(false);
        }
      } else {
        setError(error.message);
        setProgress(false);
      }
    }
```